spec/models/country_band_spec.rb

```diff
@@ -14,7 +14,7 @@
   end
 
   it "invalidates band with invalid band id" do
-    cb = CountryBand.new(number: 6, iso2: "HELLO")
-    expect(cb).to be_invalid_with_errors(number: ["is not included in the list"])
```

Comment:
Can you please keep this country-related smoke check in a separate test?

Reply:
That is already there in the previous unit test, is that fine?

Reply:
Okidoki!